dataflow/gpu-workers/tensorflow-landsat/README.md

```diff
@@ -0,0 +1,39 @@
+# Processing Landsat satellite images with GPUs
```

Comment:
Can we decouple changes to Landsat example into a separate change, or make a new folder like 'landsat_v2'?

We currently refer to Landsat example in https://cloud.google.com/dataflow/docs/samples/satellite-images-gpus , and I think these instructions will no longer work once this is merged, since Python version is no longer a parameter in this dockerfile.

Alternatively, we should merge this simultaneously with an update to https://cloud.google.com/dataflow/docs/samples/satellite-images-gpus.

Reply:
The second option may be easier, but we would need to make and test the website changes first.